presets/inference/text-generation/inference_api.py

```diff
@@ -153,7 +161,7 @@ def __post_init__(self): # validate parameters
     pipeline_kwargs["torch_dtype"] = args.torch_dtype
 
 pipeline = transformers.pipeline(
-    model_pipeline,
+    task="text-generation",
```

Comment:
If you don't use the original parameter, then consider remove the corresponding parts

Reply:
using in /chat handler